scripts/src/main/resources/scripts/command/terraform

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Check if terraform is installed.
+function doIsTerraformInstalled() {
+  if command -v terraform &> /dev/null
+  then 
+    return
+  else
+    return 255
+  fi
+}
+
+# Call Terraform with specified arguments.
+function doRun() {
+  doSetup silent
+  terraform "${@}"
+}
+
+function doSetup() {
+
+  if doIsTerraformInstalled 
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then 
+      terraform version
+    fi 
```

Comment:
This block should come at the end.
E.g. when you do `devon terraform setup` and meanwhile `TERRAFORM_VERSION` has changed, it will first print the (old) version (if it was instealld before) and then install the new version.
The result will be confusing for end users.
See other commandlets how the implement `doSetup` if unclear...

Reply:
Sorry, I don't understand.

First I asked if terraform is installed.
if yes I display the version
If not, I install it

doInstall only starting if terraform is not installed

Reply:
1. set TERRAFORM_VERSION=1.0.11
2. install terraform (`devon terraform setup`)
3. set TERRAFORM_VERSION=1.1.7
4. install/upgrade terraform (`devon terraform setup`)

Actual behaviour:
2. will not output any version and 4. will output version 1.0.11

Expected behaviour:
The version is outputted after the installation so 2. will give 1.0.11 and 4. will give 1.1.7

Reply:
I think it is now implemented in the right way. Please have a look